WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -99,9 +99,14 @@ public static void deletePeopleForLocalBlogIdExceptForFirstPage(int localTableBl
         int fetchLimit = PeopleUtils.FETCH_USERS_LIMIT;
         if (size > fetchLimit) {
             int deleteCount = size - fetchLimit;
-            String[] args = new String[]{Integer.toString(deleteCount), Integer.toString(localTableBlogId)};
-            getWritableDb().delete(PEOPLE_TABLE, "person_id " +
-                    "IN (SELECT person_id FROM people ORDER BY display_name DESC LIMIT ?) AND local_blog_id=?", args);
+            String[] args = new String[] {
+                    Integer.toString(localTableBlogId),
+                    Integer.toString(localTableBlogId),
+                    Integer.toString(deleteCount)
+            };
+            getWritableDb().delete(PEOPLE_TABLE, "local_blog_id=? AND person_id " +
```

Comment:
For future reference, you can use numbered arguments (like `?1`) in queries so you don't need to pass the same argument twice. Example:

```
String[] args = new String[] {
        Integer.toString(localTableBlogId),
        Integer.toString(deleteCount)
};
getWritableDb().delete(PEOPLE_TABLE, "local_blog_id=?1 AND person_id " +
        "IN (SELECT person_id FROM " + PEOPLE_TABLE + " WHERE local_blog_id=?1 " +
        "ORDER BY display_name DESC LIMIT ?2)", args);
```